CondCore/Utilities/o2o/deploy_offline.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh                                                                                                                                                                 
+
+home=~
+localhome=/data/condbpro
+root=/data/O2O
+cmsswroot=/cvmfs/cms.cern.ch/
+extroot=/data/ext
+release=CMSSW_10_0_5
```

Comment:
@ggovi sorry, is this intended for a 10_1_X release?

Reply:
Well it is intended for any release, depending on the deployment needs. All of the parameters of the deploy are placeholders here, and they should be properly set when the script is executed.